db/migrate/20231102114205_create_comments.rb

```diff
@@ -0,0 +1,16 @@
+class CreateComments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :comments do |t|
+      t.integer :user_id
+      t.integer :post_id
```

Comment:
- [ ] @iankiema Kindly consider removing these columns, they will be automatically generated when running your migrations using ` t.references :user, foreign_key: true,  t.references :post, foreign_key: true` code.